internal/bminventory/inventory_test.go

```diff
@@ -2619,7 +2617,7 @@ var _ = Describe("Upload and Download logs test", func() {
 		r := ioutil.NopCloser(bytes.NewReader([]byte("test")))
 		mockS3Client.EXPECT().Download(ctx, fileName).Return(r, int64(4), nil)
 		generateReply := bm.DownloadHostLogs(ctx, params)
-		downloadFileName := fmt.Sprintf("%s_%s", hostutil.GetHostnameForMsg(&host), filepath.Base(fileName))
+		downloadFileName := fmt.Sprintf("mycluster_master_%s.tar.gz", newHostID.String())
```

Comment:
Can you add a test to verify also a `bootstrap` scenario?

Reply:
Will do